core/include/macros/units.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2020 ML!PA Consulting GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     core_macros
```

Comment:
This Doxygen group is not defined. I'm surprised this was not catched by the doc static test.

None of the define below are documented and they don't show up in the generated documentation unless one knows the existence of units.h.

Reply:
> This Doxygen group is not defined:

It's defined in [`core/doc.txt`](https://github.com/RIOT-OS/RIOT/blob/master/core/doc.txt#L36)

Reply:
This is not part of the branch of this PR. A rebase would help.